tests/unit-tests/test-rest-menus.php

```diff
@@ -0,0 +1,77 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+
+use Newspack_Multibranded_Site\Taxonomy;
+use Newspack_Multibranded_Site\Meta\Menus;
+
+/**
+ * Tests editing the Url metadata from the rest api.
+ */
+class Test_Rest_Menus extends Newspack_Multibranded_Rest_Testcase {
+
+	/**
+	 * Gets a sample valid input
+	 *
+	 * @return array
+	 */
+	public function get_valid_input() {
+		return [
+			[
+				'location' => 'primary',
+				'menu'     => 1,
+			],
+			[
+				'location' => 'secondary',
+				'menu'     => 2,
+			],
+		];
+	}
+
+	/**
+	 * Test editing without permissions
+	 */
+	public function test_unauthorized() {
+		wp_set_current_user( 0 );
+		$response = $this->distpatch_request_to_edit_termmeta( Menus::get_key(), $this->get_valid_input() );
```

Comment:
```suggestion
		$response = $this->dispatch_request_to_edit_termmeta( Menus::get_key(), $this->get_valid_input() );
```

This typo is in the main branch, can be fixed in a separate PR

Reply:
will fix in another pr because also need to fix the function declaration and all other places where it's called

Reply:
fixed in https://github.com/Automattic/newspack-multibranded-site/commit/4e31e26de92d20891305ea3d6a2c1c8115d01019